<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="PlotWidget" name="v1_widget" native="true"/>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>View_1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1" colspan="3">
         <widget class="QSlider" name="v1_speed_bar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="v1_btn_add_signal">
          <property name="text">
           <string>add_signal</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+O</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="v1_btn_clone_signal2">
          <property name="text">
           <string>clone_signal_2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="v1_btn_start_pause">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>play.png</normaloff>play.png</iconset>
          </property>
          <property name="shortcut">
           <string>Space</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="v1_btn_signal_start_point">
          <property name="text">
           <string>sig_start</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="v1_speed_label">
          <property name="text">
           <string>Speed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="v1_btn_save">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (3).png</normaloff>pngwing.com (3).png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="v1_btn_link">
          <property name="text">
           <string>link</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="v1_btn_restart">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (6).png</normaloff>pngwing.com (6).png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="v1_btn_zoom_in">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>zoom_in.png</normaloff>zoom_in.png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+Up</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="v1_btn_move_left">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (5).png</normaloff>pngwing.com (5).png</iconset>
          </property>
          <property name="shortcut">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="v1_btn_zoom_out">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (1).png</normaloff>pngwing.com (1).png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+Down</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="v1_btn_signal_end_point">
          <property name="text">
           <string>sig_end</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="v1_btn_move_right">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (4).png</normaloff>pngwing.com (4).png</iconset>
          </property>
          <property name="shortcut">
           <string>Right</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="PlotWidget" name="v2_widget" native="true"/>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>View_2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="v2_btn_add_signal">
          <property name="text">
           <string>add_signal</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Shift+O</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="v2_btn_start_pause">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>play.png</normaloff>play.png</iconset>
          </property>
          <property name="shortcut">
           <string>Shift+Space</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="v2_btn_signal_start_point">
          <property name="text">
           <string>sig_start</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="v2_btn_signal_end_point">
          <property name="text">
           <string>sig_end</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="v2_btn_restart">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (6).png</normaloff>pngwing.com (6).png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="v2_btn_link">
          <property name="text">
           <string>link</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="v2_btn_move_left">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (5).png</normaloff>pngwing.com (5).png</iconset>
          </property>
          <property name="shortcut">
           <string>Shift+Left</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="v2_btn_move_right">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (4).png</normaloff>pngwing.com (4).png</iconset>
          </property>
          <property name="shortcut">
           <string>Shift+Right</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="v2_btn_save">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (3).png</normaloff>pngwing.com (3).png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="v2_btn_clone_signal1">
          <property name="text">
           <string>clone_signal_1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="v2_btn_zoom_in">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>zoom_in.png</normaloff>zoom_in.png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+Shift+Up</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="v2_btn_zoom_out">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pngwing.com (1).png</normaloff>pngwing.com (1).png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+Shift+Down</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="v2_speed_label">
          <property name="text">
           <string>Speed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QSlider" name="v2_speed_bar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
